mm.lic

```diff
@@ -0,0 +1,94 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#mm
```

Comment:
I'm not sure how I feel about using abbreviations to this degree in a script name... what do you think?

Reply:
I dont have a preference for the name. I went with something short like a command line tool. I could rename this to `;moonmage`.